### PR TITLE
[MCC-310702] Update badges in README

### PR DESCRIPTION
**Pull request by lschreck-mdsol**
This PR attempts to fix the badge links in the README because for some reason the [Public Repo Way](https://www.appveyor.com/docs/status-badges/#badges-for-projects-with-public-repositories-on-github-and-bitbucket) is not working.

After this merge, we are ready to release 2.3.0.

@mdsol/team-51 Please review/merge. Thanks!

### CODE REVIEW

**Comment by lschreck-mdsol** [comment]
@bvillanueva-mdsol Can you clarify? I don't understand that character string in your comment :confused:

File: README.md [hunk 1]
```diff
@@ -3,10 +3,12 @@
 Medidata.MAuth is a framework that provides support for authenticating web services and applications
 with the Medidata HMAC protocol, MAuth.
 
+## Build Status
+
 | Build | Status |
 | --- | --- |
-| Release | ![Release](https://ci.appveyor.com/api/projects/status/github/mauth-client-dotnet?branch=master&svg=true) |
-| Prerelease | ![Prerelease](https://ci.appveyor.com/api/projects/status/github/mauth-client-dotnet?branch=develop&svg=true) |
+| AppVeyor Release | ![Release](https://ci.appveyor.com/api/projects/status/94450nsec4kwhjpo?branch=master&svg=true) |
```
Comment (hkurniawan-mdsol) [inline README.md]: I believe this is what @bvillanueva-mdsol was talking about.
image

If there is a new pre-release or release, will that string change?
If it does, does the README need to be updated?

Comment (hkurniawan-mdsol) [inline README.md]: Sorry, didn't see your answer.

Comment (lschreck-mdsol) [inline README.md]: In the mean time I realized that and updated my comment, please see above! Thanks!

Comment (hkurniawan-mdsol) [inline README.md]: Stays the same, cool!

**Pull request merged**